File: codingjs/caughtSpeeding.js
function caughtSpeeding(speed: number, isBirthday: boolean): number {
  const addSpeedLimit = isBirthday ? 5 : 0;
  if (speed <= 60 + addSpeedLimit) return 0;
  if (speed <= 80 + addSpeedLimit) return 1;
  return 2;
}

File: codingbat/java/caughtSpeeding.java
public int caughtSpeeding(int speed, boolean isBirthday) {
  int addHigher = isBirthday ? 5 : 0;
  if (speed >= 81 + addHigher) return 2;
  if (speed >= 61 + addHigher) return 1;
  return 0;
}

